src/client/src/ui/analytics/views/pnlBar.jsx

```diff
@@ -22,12 +22,13 @@ export default class PNLBar extends React.Component{
     let indicatorClass = 'analytics__barchart-indicator ';
     indicatorClass += isPositive ? 'analytics__barchart-indicator--positive' : 'analytics__barchart-indicator--negative';
     let amountStr = numeral(baseValue).format();
+    let ccyPrefix = this.props.isPnL ? this.props.model.symbol.substr(0, 3) : '';
```

Comment:
We do a lot of substring calls on symbol. Can we not model this correctly in the model?  So a proper CurrencyPair object with base and terms properties?